app/controllers/auth_tokens_controller.rb

```diff
@@ -25,7 +25,7 @@ def create
     end
 
     @auth_token = AuthToken.new(
-      { user_agent: request.headers[:'user-agent'] }
+      { user_agent: request.headers[:'user-agent'], application: params[:application] }
```

Comment:
This shouldn't be necessary, I think? This should/could be part of the permitted_attributes below?

Reply:
no clue why i added this here 🤦